website/events/models/event.py

```diff
@@ -247,6 +247,41 @@ def cancellation_allowed(self):
             and self.registration_start <= now < self.start
         )
 
+    @property
+    def present_registrations(self):
+        return self.participants.filter(present=True)
+
+    @property
+    def unpresent_registrations(self):
+        return self.participants.filter(present=False)
+
+    @property
+    def unpaid_registrations(self):
+        return (
+            self.participants.filter(payment__isnull=True)
+            if self.payment_required
+            else None
+        )
+
+    @property
+    def unpaid_present_registrations(self):
+        return (
+            self.present_registrations.filter(payment__isnull=True)
+            if self.payment_required
+            else None
+        )
+
+    @property
+    def everything_paid(self):
```

Comment:
```suggestion
    def everyone_paid(self):
```

It's about everyone who is participating right?